src/Features/CSharpTest/ExtractMethod/ExtractLocalFunctionTests.cs

```diff
@@ -5445,9 +5437,9 @@ namespace C
 
         [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsExtractLocalFunction)]
         [WorkItem("https://github.com/dotnet/roslyn/issues/45422")]
-        public async Task TestMissingOnExtractLocalFunction()
+        public async Task TestOnExtractLocalFunction()
```

Comment:
this was hte case that used to fail before because the rewrite would end up rewriting the node the local function wanted to go into.  This scenario used to fail because we'd detect and bail out.

However, now that we properly pick the right plae to put things, these tests now actually work.  So win/win.